validator/accounts/v2/export_test.go

```diff
@@ -0,0 +1,117 @@
+package v2
+
+import (
+	"context"
+	"flag"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	"github.com/urfave/cli/v2"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	v2 "github.com/prysmaticlabs/prysm/validator/keymanager/v2"
+)
+
+func setupWallet(t *testing.T, testDir string) *Wallet {
+	walletDir := filepath.Join(testDir, walletDirName)
+	passwordsDir := filepath.Join(testDir, passwordDirName)
+	ctx := context.Background()
+	err := initializeDirectWallet(walletDir, passwordsDir)
+	assert.NoError(t, err)
+	cfg := &WalletConfig{
+		WalletDir:      walletDir,
+		PasswordsDir:   passwordsDir,
+		KeymanagerKind: v2.Direct,
+	}
+	w, err := NewWallet(ctx, cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.NoError(t, err)
+	keymanager, err := w.InitializeKeymanager(ctx, true)
+	if err != nil {
+		t.Fatalf("Could not initialize keymanager: %v", err)
+	}
+
+	assert.NoError(t, err)
+	if _, err := keymanager.CreateAccount(ctx, password); err != nil {
+		t.Fatalf("Could not create account in wallet: %v", err)
+	}
+
+	assert.NoError(t, err)
+	return w
+}
+
+func TestZipAndUnzip(t *testing.T) {
+	testDir := testutil.TempDir()
+	walletDir := filepath.Join(testDir, walletDirName)
+	passwordsDir := filepath.Join(testDir, passwordDirName)
+	exportDir := filepath.Join(testDir, exportDirName)
+	importDir := filepath.Join(testDir, importDirName)
+	defer func() {
+		assert.NoError(t, os.RemoveAll(walletDir))
+		assert.NoError(t, os.RemoveAll(passwordsDir))
+		assert.NoError(t, os.RemoveAll(exportDir))
+		assert.NoError(t, os.RemoveAll(importDir))
+	}()
+	wallet := setupWallet(t, testDir)
+
+	accounts, err := wallet.AccountNames()
+	assert.NoError(t, err)
+
+	if len(accounts) == 0 {
```

Comment:
You can do require.Equal(t, len(accounts), 0)

Reply:
The order of arguments is `expected, got` in equal. So, something like:
```golang
require.Equal(t, 0, len(accounts), "Expected more accounts") // the last param is optional
```
Please see my comment on https://github.com/prysmaticlabs/prysm/pull/6626 for my rationale.

Reply:
Dismissing my suggestion (as it is too fragile to require the order, and not that necessary): https://github.com/prysmaticlabs/prysm/pull/6631